### ISSUE TITLE
[FR] Allow to highlight/star/set priority/value for tabs

### ISSUE THREAD

**Issue by icytree** [mbnuqw/sidebery#902]
## Description
Along the line of this [feature request](https://github.com/mbnuqw/sidebery/issues/666#issuecomment-1381133734) regarding quickly saving information about open tabs for later use, I thought it could be very useful to be able to set some level of priority/value perhaps to indicate tabs that are more important (and worth coming back to, when querying X subject). Being able to set this and even cycle through the levels of priority/value with hotkeys would also make this awesome.

I guess it could be currently integrated in a simpler form using the bookmarks by providing a setting to display tabs that are bookmarked differently (ex. highlighted in Y color).

**Comment by emvaized** [comment]
[Tab rename](https://github.com/mbnuqw/sidebery/issues/185) feature will be fine for this purpose, once it's ready

**Comment by koppor** [comment]
> I thought it could be very useful to be able to set some level of priority/value perhaps to indicate tabs that are more important (and worth coming back to, when querying X subject).

In a very basic form, that is possible using coloring:

![image](https://user-images.githubusercontent.com/1366654/220850058-637f724d-d8b4-42f9-a641-7f448a839daa.png)

**Comment by icytree** [comment]
Good point @koppor ! The only thing is that I would love if it could be bound to a keybinding for ease of use. But it would indeed be great way for setting a value hierarchy.
